src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -194,7 +185,7 @@ CHIP_ERROR AddCommonTxtElements(const BaseAdvertisingParams<Derived> & params, c
 CHIP_ERROR DiscoveryImplPlatform::Advertise(const CommissionAdvertisingParameters & params)
 {
     CHIP_ERROR error = CHIP_NO_ERROR;
-    MdnsService service;
+    DnssdService service;
```

Comment:
This can be done in a separate PR, but imho these structures shouldn't have the Dnssd prefix as they are already part of the `Dnssd` namespace.